src/com/opc_ua/opcua_local_handler.cpp

```diff
@@ -161,8 +161,6 @@ void COPC_UA_Local_Handler::generateServerStrings(TForteUInt16 paUAServerPort, U
   paServerStrings.mHostname = std::string("opc.tcp://");
 #ifdef FORTE_COM_OPC_UA_CUSTOM_HOSTNAME
   paServerStrings.mHostname.append(FORTE_COM_OPC_UA_CUSTOM_HOSTNAME);
-  paServerStrings.mHostname.append("-"s);
-  paServerStrings.mHostname.append(helperBuffer);
```

Comment:
as the helperBuffer is not being used under this #ifdef anymore, you could move it inside the #else

Reply:
oh, I missed that. Great, then let's merge this!